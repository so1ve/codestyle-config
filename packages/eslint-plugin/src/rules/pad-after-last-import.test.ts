import { run } from "./_test";
import rule, { RULE_NAME } from "./pad-after-last-import";

const valid = ['import a from "foo";\n\nconst b = 1;'];
const invalid = [
	'import a from "foo";\nconst b = 1;',
	'import a from "foo";\n// comment\nconst b = 1;',
];

run({
	name: RULE_NAME,
	rule,
	valid,
	invalid,
});
