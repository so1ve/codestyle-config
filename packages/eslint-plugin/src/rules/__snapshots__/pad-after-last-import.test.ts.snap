// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`pad-after-last-import > invalid > Invalid #0: import a from "foo";
const b = 1; 1`] = `
"import a from "foo";

const b = 1;"
`;

exports[`pad-after-last-import > invalid > Invalid #1: import a from "foo";
// comment
const b = 1; 1`] = `
"import a from "foo";

// comment
const b = 1;"
`;
